src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,455 @@
+    examples:
+        - name: Gets the list of AuthorizatioRule by Namespace
+          text: helps['sb namespace authorizationrule get --resource-group {rg} --namespace-name {namespacename}']
+
+    """
+
+helps['sb namespace authorizationrule listkeys'] = """
+    type: command
+    short-summary: Gets the connectionstriongs of AuthorizatioRule for the namespace.  
+    examples:
+        - name: Gets the connectionstriongs of AuthorizatioRule for the namespace.
+          text: helps['sb namespace authorizationrule listkeys --resource-group {rg} --namespace-name {namespacename}
+           --name {authoname}']
+
+    """
+
+helps['sb namespace authorizationrule regenerate_keys'] = """
+    type: command
+    short-summary: Regenerate the connectionstriongs of AuthorizatioRule for the namespace.  
+    examples:
+        - name: Regenerate the connectionstriongs of AuthorizatioRule for the namespace.
+          text: helps['sb namespace authorizationrule regenerate_keys --resource-group {rg}
+           --namespace-name {namespacename} --name {authoname} --regeneratekey {primary}']
+
+    """
+
+helps['sb namespace authorizationrule delete'] = """
+    type: command
+    short-summary: Deletes the AuthorizatioRule of the namespace.  
+    examples:
+        - name: Deletes the AuthorizatioRule of the namespace. 
+          text: helps['sb namespace authorizationrule delete --resource-group {rg} --namespace-name {namespacename}
+           --name {authoname}']
+
+    """
+
+helps['sb queue create'] = """
+    type: command
+    short-summary: Creates the ServiceBus Queue
+    examples:
+        - name: Create a new queue.
+          text: helps['sb queue create --resource-group {rg} --namespace-name {namespacename} --name {queuename}']
+
+    """
+
+helps['sb queue get'] = """
+    type: command
+    short-summary: Get the Queue Details
+    examples:
+        - name: Get the Queue details.
+          text: helps['sb queue get --resource-group {rg} --namespace-name {namespacename} --name {queuename}']
+
+    """
+
+helps['sb queue list'] = """
+    type: command
+    short-summary: List the Queueby Namepsace
+    examples:
+        - name: Get the Queues by Namespace.
+          text: helps['sb queue list --resource-group {rg} --namespace-name {namespacename}']
+
+    """
+
+helps['sb queue delete'] = """
+    type: command
+    short-summary: Deletes the Queue
+    examples:
+        - name: Deletes the queue
+          text: helps['sb queue delete --resource-group {rg} --namespace-name {namespacename} --name {queuename}']
+
+    """
+
+helps['sb queue authorizationrule create'] = """
+    type: command
+    short-summary: Creates Authorization rule for the given Queue 
+    examples:
+        - name: Creates Authorization rules
+          text: helps['sb queue authorizationrule create --resource-group {rg} --namespace-name {namespacename}
+           --queue-name {queuename}
+           --name {authoname} --access-rights [Send, Listen]']
```

Comment:
`--access-rights [Send, Listen]`
Choose one for the example.
Currently, this won't actually execute.